oozie-to-airflow/utils/el_utils.py

```diff
@@ -93,15 +94,19 @@ def parse_el_func(el, el_func_map=EL_FUNCTIONS):
     if not fn_match:
         return None
 
-    # fn_match is of the form [('concat', "'ls', '-l'")]
-    # for an el function like ${concat('ls', '-l')}
-    if fn_match[0][0] not in el_func_map:
-        raise KeyError('{} EL function not supported.'.format(fn_match[0][0]))
+    parsed_fns = []
+    for fn in fn_match:
+        # fn_match is of the form [('concat', "'ls', '-l'")]
+        # for an el function like ${concat('ls', '-l')}
+        if fn[0] not in el_func_map:
+            raise KeyError('{} EL function not supported.'.format(fn[0]))
 
-    mapped_func = el_func_map[fn_match[0][0]]
+        mapped_func = el_func_map[fn_match[0][0]]
```

Comment:
I might not be understanding this correctly, but it looks like we're iterating through fn_match, but each time we're only using fn_match[0][0]. When not fn[0] instead?

At the surface level it seems like this can potentially yield incorrect behavior.